package.json

```diff
@@ -5,7 +5,11 @@
   "main": "index.js",
   "scripts": {
     "test": "mocha --require esm",
-    "tw": "mocha --watch --watch-files 'test/**/*.js' --watch-files 'src/**/*.js' --reporter dot --require esm"
+    "tw": "mocha --watch --watch-files 'test/**/*.js' --watch-files 'src/**/*.js' --reporter dot --require esm",
```

Comment:
Now that you know how to run one npm script from another, how could you modify the `tw` script to make use of this feature?